django/applications/catmaid/control/classification.py

```diff
@@ -210,18 +211,17 @@ class classification_project (which lives in dummy project -1) and links to
 
     return cici_q
 
-def get_classification_roots_view(request, workspace_pid, project_id, cursor=None):
-    """
-    """
+def get_classification_roots_view(request, workspace_pid, project_id, cursor=None) -> None:
+    # FIXME: Appears to be unused. Does not return anything.
     links_q = get_classification_links_qs( workspace_pid, project_id, cursor )
 
 
-def get_classification_number( project_id ):
+def get_classification_number(project_id) -> int:
     """ Returns the number of classification graphs, linked to a
     project.
     """
     # Get all links
-    links_q = get_classification_links_qs( project_id )
+    links_q = get_classification_links_qs( project_id ) # FIXME: Needs a workspace argument before this
```

Comment:
Indeed, both the `get_classification_number()` signature and the `get_classification_links_qs()` call lack a `workspace_id` argument. The URL definition in `urls.py` linking to this view function does provide it already. However, to be a proper function based view, this function also lacks a `request` parameter (and a permissions decorator). The front-end doesn't make use of this API at the moment and I feel it would also be okay to just remove it.